common/src/main/java/org/jboss/eap/qe/ts/common/docker/DockerContainers.java

```diff
@@ -0,0 +1,23 @@
+package org.jboss.eap.qe.ts.common.docker;
+
+public class DockerContainers {
```

Comment:
JavaDoc missing.

Reply:
Makes sense. I will add description what is the purpose of this class.